src/components/IconDetail.vue

```diff
@@ -39,6 +39,125 @@ const copy = async(type: string) => {
   }, 2000)
 }
 
+const copyPng = async () => {
+	console.log(color.value);
+	let svg = await getIconSnippet(props.icon, "svg", true, color.value);
+	console.log(svg);
+	let containerDiv = document.createElement("div");
+	containerDiv.innerHTML = svg;
+	//Now we have a DOM element:
+	svg = containerDiv.querySelector("svg");
+	svg.setAttribute("style", `color: ${color.value};`);
+	let data = new XMLSerializer().serializeToString(svg);
+	let svgBlob = new Blob([data], {
+		type: "image/svg+xml;charset=utf-8",
+	});
+	let src = URL.createObjectURL(svgBlob);
+	let image = await toPng(src);
+	console.log(image);
+	navigator.clipboard
+		.write([
+			new ClipboardItem({
+				"image/png": toBlob(image),
+			}),
+		])
+		.then(
+			(...a) => (
+				(copied.value = "Copied"),
+				setTimeout(() => (copied.value = false), 2000)
+			),
+			(...a) => (
+				(copied.value = "Failed to copy"),
+				setTimeout(() => (copied.value = false), 2000)
+			)
+		);
+	function toBlob(dataurl) {
+		var arr = dataurl.split(","),
+			mime = arr[0].match(/:(.*?);/)[1],
+			bstr = atob(arr[1]),
```

Comment:
`atob` could be unreliable in Node.js environments, I don't know enough about this project to understand all the environments, is it ok to use here?

Reply:
Using `fetch(dataurl).then(res => res.blob())` instead